lib/TopBar/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Grid from 'react-bootstrap/lib/Grid';
+import Row from 'react-bootstrap/lib/Row';
+import cx from 'classnames';
+
+const TopBar = props => {
+  const fixedClass = cx({ 'top-bar__wrapper--fixed': props.fixed });
+  return (
+    <Grid className="top-bar" fluid>
+      <div className={`top-bar__wrapper ${fixedClass}`}>
+        <Row className="top-bar__inner">{props.children}</Row>
+      </div>
+    </Grid>
+  );
+};
+
+TopBar.propTypes = {
+  fixed: PropTypes.bool,
+  children: PropTypes.shape()
```

Comment:
Does children accept an object or is it a node/array of nodes? Your default prop for this sets it as an array so I believe this needs changing 👍